src/language-server/engine/index.ts

```diff
@@ -87,7 +87,10 @@ export class ApolloEngineClient extends GraphQLDataSource {
         fieldStats.get(parentType) ||
         fieldStats.set(parentType, new Map()).get(parentType)!;
 
-      fieldsMap.set(fieldName, fieldStat.metrics.fieldHistogram.durationMs);
+      fieldsMap.set(
+        fieldName,
+        fieldStat.metrics.fieldHistogram.durationMs ?? null
```

Comment:
I believe we can add `avoidOptionals` to the config to avoid this change? I believe the actual type returned here will not be undefined